cli/app/app.go

```diff
@@ -234,18 +234,22 @@ func ProjectDelete(p project.APIProject, c *cli.Context) error {
 		RemoveVolume: c.Bool("v"),
 	}
 	if !c.Bool("force") {
-		options.BeforeDeleteCallback = func(stoppedContainers []string) bool {
```

Comment:
I think if `ListStoppedContainers` is exposed then the same functionality is still available. A library user can call `ListStoppedContainers` and then just perform and "before" operations before making the call to `Delete`.